curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/delete-many-documents-with-model.remove.english.md

```diff
@@ -6,13 +6,13 @@ challengeType: 2
 
 ## Description
 <section id='description'>
-Model.remove() is useful to delete all the documents matching given criteria. Delete all the people whose name is “Mary”, using Model.remove(). Pass it to a query document with the “name” field set, and of course a callback.
-Note: Model.remove() doesn’t return the deleted document, but a JSON object containing the outcome of the operation, and the number of items affected. Don’t forget to pass it to the done() callback, since we use it in tests.
+<code>Model.remove()</code> is useful to delete all the documents matching given criteria. 
 </section>
 
 ## Instructions
 <section id='instructions'>
-
+Delete all the people whose name is “Mary”, using <code>Model.remove()</code>. Pass it to a query document with the <code>name</code> field set, and of course a callback.
+<strong>Note:</strong> <code>Model.remove()</code> doesn’t return the deleted document, but a JSON object containing the outcome of the operation, and the number of items affected. Don’t forget to pass it to the <code>done()</code> callback, since we use it in tests.
```

Comment:
the code tag is pushing that sentence down to it's own line and leaving a large space (photo below)- I would like to see the sentence start on the same line as the note to be consistent with other notes in these lessons - This can be ignored as well if you want @RandellDawson,  maybe adding another word at the beginning without the `<code>` tag would keep that inline - something like `The <code>Model.remove()</code> function doesn't`...
<img width="818" alt="Screen Shot 2019-03-09 at 7 52 11 AM" src="https://user-images.githubusercontent.com/20648924/54072444-82203600-4240-11e9-9a15-ce7ce74e4fb1.png">